stdlib/public/Darwin/AppKit/NSGraphics.swift

```diff
@@ -83,13 +84,14 @@ extension Sequence where Iterator.Element == NSRect {
   public func clip() {
     precondition(NSGraphicsContext.current != nil,
                  "There must be a set current NSGraphicsContext")
-    let rects = Array(self)
-    let count = rects.count
-    guard count > 0 else { return }
-    rects.withUnsafeBufferPointer { rectBufferPointer in
-      guard let rectArray = rectBufferPointer.baseAddress else { return }
-      __NSRectClipList(rectArray, count)
+    func _clip(buffer: UnsafeBufferPointer<NSRect>) {
+      guard let bufferAddress = buffer.baseAddress, !buffer.isEmpty else {
+				return
+      }
+      __NSRectClipList(bufferAddress, buffer.count)
     }
+    withContiguousStorageIfAvailable(_clip) ??
```

Comment:
Minor stylistic nitpick. Just discussed it with @milseman and we agreed that it would be more readable if these two lines are reformatted as the following:

```swift
withContiguousStorageIfAvailable(_clip)
?? Array(self).withUnsafeBufferPointer(_clip)
```

This way it is clear that the second line is not a new statement, but instead is a previous expression continued. 
What do you think?

Reply:
Sounds good. Updated.